docs/user-guide/computation.ipynb

```diff
@@ -240,16 +240,16 @@
     "    {'a': sc.Variable(dims=['x', 'y'], values=np.random.rand(2, 3)),\n",
     "     'b': sc.Variable(dims=['y', 'x'], values=np.random.rand(3, 2)),\n",
     "     'c': sc.Variable(dims=['x'], values=np.random.rand(2)),\n",
-    "     'd': sc.Variable(1.0)},\n",
+    "     'd': sc.Variable(value=1.0)},\n",
     "    coords={\n",
-    "        'x': sc.Variable(['x'], values=np.arange(2.0), unit=sc.units.m),\n",
-    "        'y': sc.Variable(['y'], values=np.arange(3.0), unit=sc.units.m)})\n",
+    "        'x': sc.Variable(dims=['x'], values=np.arange(2.0), unit=sc.units.m),\n",
+    "        'y': sc.Variable(dims=['y'], values=np.arange(3.0), unit=sc.units.m)})\n",
     "d2 = sc.Dataset(\n",
     "    {'a': sc.Variable(dims=['x', 'y'], values=np.random.rand(2, 3)),\n",
     "     'b': sc.Variable(dims=['y', 'x'], values=np.random.rand(3, 2))},\n",
     "    coords={\n",
-    "        'x': sc.Variable(['x'], values=np.arange(2.0), unit=sc.units.m),\n",
-    "        'y': sc.Variable(['y'], values=np.arange(3.0), unit=sc.units.m)})"
+    "        'x': sc.Variable(dims=['x'], values=np.arange(2.0), unit=sc.units.m),\n",
+    "        'y': sc.Variable(dims=['y'], values=np.arange(3.0), unit=sc.units.m)})"
    ]
```

Comment:
We had originally postponed the rollout to docs until we were sure this is what we wanted to, so I'd say:
- Make sure our current discussion in `sc.array` vs. `sc.DataArray` is concluded.
- Don't change in this PR, do it as follow-up, since this is big already.